packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -227,6 +224,19 @@ std::vector<std::vector<Scalar> > generate_fem1d_element_values() {
   return mat;
 }
 
+template<class Scalar, class Node>
+Kokkos::View<Scalar*[2], Kokkos::LayoutLeft, typename Node::device_type > generate_fem1d_element_values_kokkos() {
+  Kokkos::View<Scalar*[2], Kokkos::LayoutLeft, typename Node::device_type > mat;
+
+  Kokkos::resize(mat, 2);
+  //plz don't abuse UVM
```

Comment:
Well, then don't abuse UVM!  You're welcome to fill on host, but then do this:
```c++
auto mat_h = Kokkos::create_mirror_view (mat);
mat_h(0,0) = 1.0;
// ... etc. ...
Kokkos::deep_copy (mat, mat_h);
return mat;
```

Reply:
Also, don't assign integers to Scalar.  That only works if Scalar is `double`, etc.  It breaks if Scalar is `complex`.